app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
+  has_many :likes
```

Comment:
- [ ] Well done setting up the associations between models, I recommend to add the foreign key for likes.

`has_many :likes, foreign_key: 'author_id'`